build_pbf_glyphs/src/main.rs

```diff
@@ -33,14 +33,29 @@ use std::sync::atomic::{AtomicUsize, Ordering};
 use std::thread;
 use std::time::Instant;
 
-use clap::{command, crate_authors, crate_description, crate_version, Arg};
+use clap::{command, Parser};
 use pbf_font_tools::freetype::{Face, Library};
 use pbf_font_tools::{get_named_font_stack, glyph_range_for_face, Glyphs};
 use protobuf::{CodedOutputStream, Message};
 use spmc::{channel, Receiver};
 
 static TOTAL_GLYPHS_RENDERED: AtomicUsize = AtomicUsize::new(0);
 
+#[derive(Parser, Debug)]
+#[command(version, author, about, long_about = None)]
```

Comment:
why do you need `long_about = None` ?

Reply:
Because I misread the docs, apparently 😂 Removing...